stable/metrics-server/templates/metric-server-service.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     chart: {{ template "metrics-server.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+{{- with .Values.serviceLabels }}
```

Comment:
As `.Values.service.annotations` has been merged, this should probably be changed to `.Values.service.labels` for consistency.